### PR TITLE
Release artifacts for v1.1.0

### PR DESCRIPTION
**Pull request by erikfuller**
Release artifacts for v1.1.0

### CODE REVIEW

File: files/controller-installation/deploy-v1.0.3.yaml
Comment (mikestvz) [inline files/controller-installation/deploy-v1.0.3.yaml:1]: Do we need to delete this one?

Comment (erikfuller) [reply files/controller-installation/deploy-v1.0.3.yaml:1]: There's logic in the `gitub-release.sh` script that only keeps the last five of these files. I'm not really sure there's ongoing value in keeping these older files around but there isn't an urgent need to remove them either. 

These files will continue to be available via version tags, for example: https://github.com/aws/aws-application-networking-k8s/tree/v1.0.7/files/controller-installation

Comment (erikfuller) [reply files/controller-installation/deploy-v1.0.3.yaml:1]: The lines in the script that remove old YAMLS: https://github.com/aws/aws-application-networking-k8s/blob/main/scripts/github-release.sh#L73-L74

**Review by mikestvz — APPROVED** [review]
/LGTM

**Pull request merged**